packages/example/utils/convert.ts

```diff
@@ -0,0 +1,12 @@
+import { Chain } from '@chain-registry/types';
+import { ChainRegistry } from '@cosmos-kit/core';
+
+// TODO discuss Chain 
+// maybe simplify so we can use `Chain` throughout the app
+export function convert(chain: Chain): ChainRegistry {
+    return {
+      name: chain.chain_name,
+      active: false,
+      raw: chain,
```

Comment:
So I suggest we use `Chain` from `@chain-registry/types` and if we need to keep track of `active`, we can make a new data structure stored in the wallet, something like, `activeChains`:

```js
export class WalletClass {
    // key is chain.chain_name (this could also be Map too if you prefer)
    enabledChains: Record<string, boolean>;
}
```

Reply:
this will allow us not to wrap or `convert()` `Chain` from chain-registry data.

Reply:
as we discussed, we'll rename it. but `covert` will still in use.